onnxruntime/core/mlas/lib/qgemm.h

```diff
@@ -73,7 +73,7 @@ MlasGemmQuantTryGemvKernel(
 }
 
 template <typename KernelType>
-MLAS_FORCEINLINE
+MLAS_FORCEINLINE constexpr
```

Comment:
any reason to add constexpr here? ZeroPointA  is not known at compiling time

Reply:
similar to other functions

Reply:
Here constexpr means if the arguments are known at compile time, the result would be too.

Reply:
yes, but as i mentioned, ZeroPointA is not known at compiling time. It is meaningless. If it is for some style enforcement, that's fine to have it.